decide_panel/src/components/Graphics.js

```diff
@@ -0,0 +1,97 @@
+import { useEffect, useState } from "react";
+import Api from "../services/backend";
+import { ProgressBar } from "primereact/progressbar";
+import React from "react";
+import { Pie } from "react-chartjs-2";
+import Chart from 'chart.js/auto';
+
+
+const Graphics = () => {
+  const [errorConection, setErrorConection] = useState(null);
+  const [state, setState] = useState({
+    data: '{}',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
    data: "{}",
```